docker/Dockerfile.ci

```diff
@@ -8,7 +8,7 @@ RUN sudo yum upgrade -y \
     && sudo rm -rf /var/cache/yum
 
 # specify all versions to be installed, partial versions also understood
-ARG RUBY_VERSIONS="2.3 2.6"
+ARG RUBY_VERSIONS="2.6 2.7"
```

Comment:
EOL?

Reply:
Actually, these lines apparently don't do anything :thinking: 
Because I can still see this in the logs:
```
=== [rbenv4ci] Switching to 2.3.8
=== [rbenv4ci] Bundling /home/ruby/app/Gemfile on ruby 2.3.8p459 (2018-10-18 revision 65136) [x86_64-linux] with Bundler version 1.17.3

=== [rbenv4ci] Switching to 2.6.1
=== [rbenv4ci] Bundling /home/ruby/app/Gemfile on ruby 2.6.1p33 (2019-01-30 revision 66950) [x86_64-linux] with Bundler version 1.17.3
```

So I think I'll actually remove this commit for now.